standard/ERCs/erc-6900.md

```diff
@@ -687,27 +567,23 @@ The `executeFromModuleExternal` function MUST allow modules to call external add
+This proposal includes several interfaces that build on ERC-4337. First, we standardize a set of modular functions that allow smart contract developers greater flexibility in bundling validation, execution, and hook logic. We also propose interfaces that provide methods for querying execution functions, validation functions, and hooks on a modular account. The rest of the interfaces describe a module's methods for exposing its modular functions and desired configuration, and the modular account's methods for installing and removing modules and allowing execution across modules and external addresses.
 
 ## Backwards Compatibility
 
-No backward compatibility issues found.
+TODO
 
 ## Reference Implementation
 
 See `https://github.com/erc6900/reference-implementation`
 
 ## Security Considerations
 
-The modular smart contract accounts themselves are trusted components. Installed plugins are trusted to varying degrees, as plugins can interact with an arbitrarily large or small set of resources on an account. For example, a wide-reaching malicious plugin could add reverting hooks to native function selectors, bricking the account, or add execution functions that may drain the funds of the account. However, it is also possible to install a plugin with a very narrow domain, and depend on the correctness of the account behavior to enforce its limited access. Users should therefore be careful in what plugins to add to their account.
-
-Users should perform careful due diligence before installing a plugin and should be mindful of the fact that plugins are potentially dangerous. The plugin's manifest can give users an understanding of the domain of the plugin, i.e., the requested permissions to install certain validation functions and/or hooks on certain execution selectors. Generally, plugins that include native function selectors in their domain, e.g., plugins that add a validation hook to the native `uninstallPlugin()` function, can introduce significantly more harm than plugins that simply add validation hooks to function selectors that the plugin itself is adding to the account.
+The modular smart contract accounts themselves are trusted components. Installed modules are trusted to varying degrees, as modules can interact with an arbitrarily large or small set of resources on an account. For example, a wide-reaching malicious module could add reverting hooks to native function selectors, bricking the account, or add execution functions that may drain the funds of the account. However, it is also possible to install a module with a very narrow domain, and depend on the correctness of the account behavior to enforce its limited access. Users should therefore be careful in what modules to add to their account.
```

Comment:
Do you think it makes sense to mention that users have the purview to dictate permissions for installed modules at a granular level, separate from the module itself.

What I mean is, you could install a module with a standard set of hooks you use for *every* module (with varying configurations), so even a malicious module has to go through hooks it can't do anything about. This is a super strong security model-- especially if the wallet service itself appends these hooks separate from the applications users connect to.

Reply:
Yes, this is worth mentioning. Do you want to take a stab at writing up this consideration?

Reply:
Yup, will do!

Reply:
Actually, could we do this on a stacked PR? To be able to close this & work off of `develop`?